8/run-client-local.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 export ADVERTISEDIP=`ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v -m1 '127.0.0.1'`
 export GATEWAYPORT=3001
```

Comment:
Do we need the advertised ip and gateway port since we're using persistent membership table?

Reply:
Oh leftovers